tests/baselines/reference/allowImportClausesToMergeWithTypes.js

```diff
@@ -45,7 +45,7 @@ b_1["default"];
 "use strict";
 exports.__esModule = true;
 var x = { x: "" };
-zzz;
+a_1["default"];

```

Comment:
This change looks weird, but it is correct. However, there should be an `import a_1 = require("./a")` here, which is elided because of another bug: https://github.com/microsoft/TypeScript/issues/50455